test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -797,6 +797,32 @@ public void WorkspaceContextMenu_IfSubmenuOpenOnMouseHover()
             Assert.IsTrue(currentWs.WorkspaceLacingMenu.IsSubmenuOpen);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void TestDumpLibraryToXml()
+        {
+            ViewModel.DumpLibraryToXmlCommand.Execute(null);
+
+            // Look for library dump files aged within 10 seconds from the current time
+            var time = System.DateTime.Now;
+            var timeout = 10;
+
+            for (var t = 0; t < timeout; t++)
+            {
+                string fileName = string.Format("LibrarySnapshot_{0}.xml", time.ToString("yyyyMMddHmmss"));
+                string fullFileName = Path.Combine(Model.PathManager.LogDirectory, fileName);
+                Console.WriteLine("Looking for library dump file at " + fullFileName);
+                if (File.Exists(fullFileName))
+                {
+                    File.Delete(fullFileName);
```

Comment:
Can we redirect the XML file to the folder indicated by `SystemTestBase.TempFolder`? I believe that folder is randomly generated and will always be destroyed towards the end of the test. My point is, we may not even need to delete the output file manually here.